corehq/apps/users/role_utils.py

```diff
@@ -126,3 +128,24 @@ def enable_attendance_coordinator_role_for_domain(domain):
     if role.is_archived:
         role.is_archived = False
         role.save()
+
+
+def get_commcare_analytics_access_for_user_domain(couch_user, domain):
+    if not SUPERSET_ANALYTICS.enabled(domain):
+        return None
+
+    domain_membership = couch_user.get_domain_membership(domain)
+    is_admin = domain_membership.is_admin
+
+    domain_role_permissions = domain_membership.role.permissions
+    permissions = {
+        "can_edit": True if is_admin else domain_role_permissions.edit_commcare_analytics,
```

Comment:
Nit: These can be shortened to `is_admin or domain_role_permissions.edit_commcare_analytics`.

Reply:
True!